src/Analyzers/CSharp/CodeFixes/ConvertNamespace/ConvertNamespaceTransform.cs

```diff
@@ -156,11 +149,112 @@ public static (SourceText text, TextSpan semicolonSpan) ConvertNamespaceDeclarat
             return new TextChange(new TextSpan(textLine.Start, commonIndentation), newText: "");
         }
 
+        private static SourceText IndentNamespace(
+            ParsedDocument document, string indentation, SyntaxAnnotation annotation, CancellationToken cancellationToken)
+        {
+            var syntaxTree = document.SyntaxTree;
+            var text = document.Text;
+            var root = document.Root;
+
+            var blockScopedNamespace = (NamespaceDeclarationSyntax)root.GetAnnotatedNodes(annotation).Single();
+            var openBraceLine = text.Lines.GetLineFromPosition(blockScopedNamespace.OpenBraceToken.SpanStart).LineNumber;
+            var closeBraceLine = text.Lines.GetLineFromPosition(blockScopedNamespace.CloseBraceToken.SpanStart).LineNumber;
+
+            using var _ = ArrayBuilder<TextChange>.GetInstance(out var changes);
```

Comment:
nit: consider initializing with the nubmer of lines intended to be indented.  will prevent the need to do any resizing.

Reply:
➡️ Updated